### ISSUE TITLE
Support for regressions 

### ISSUE THREAD

**Issue by igorvere** [HDI-Project/ATM#72]
Hello, If I understood correctly currently ATM solves only classification tasks. I wonder if there is a plan for adding support for regression problems. Thanks

**Comment by bcyphers** [comment]
Yes! The plan is to eventually add support for other kinds of models, starting with regressions. However, this will come a little bit down the road -- if you'd like the functionality soon, you might have to fork the project and build it yourself. If you do, feel free to open up a PR and contribute!
